src/components/Database/DataViewer/CloneDialog.tsx

```diff
@@ -63,19 +67,33 @@ const cloneKey = (key: string, ref: firebase.database.Reference) => {
 
 export const CloneDialog = React.memo<Props>(function CloneDialog$({
   realtimeRef,
+  queryParams,
   onComplete,
 }) {
   ok(realtimeRef.parent, 'Cannot clone the root node!');
 
+  // queries only exist if queryParams are passed into the dialog
+  let query =
```

Comment:
I think this is:

```js
let query = queryParams ?? applyQuery(realtimeRef, queryParams);
```

Reply:
Removed applyQuery and now we just pass the query.